### DISCUSSION TITLE
Welches tag für ein Vereinsheim im Schrebergarten?

### DISCUSSION THREAD

**Discussion by biketeur** [streetcomplete/StreetComplete#3566] [Q&A]
In vielen Kleingartenanlagen gibt es ein Vereinsheim, meist mit Gastronomie.
Welches tag sollte man für diese Gebäudeart verwenden?
Vielen Dank für eine Antwort.

**Comment by matkoniecz** [comment]
Seems that "retail" (applying also to restaurants and similar services) would fit

Original: Attaching a photo of such building would help.

**Reply by HolgerJeromin** [reply]
https://www.google.com/search?q=Vereinsheim+im+Schrebergarten&tbm=isch
shows many.

**Reply by matkoniecz** [reply]
Seems that "retail" (applying also to restaurants and similar services) would fit
